app/templates/pains_verify_and_coefficient_use.html

```diff
@@ -1,120 +1,143 @@
-          if (include_mpo == "True") {
-            document.write('Mpo Score 0 Color: <input type="color" name="mpo0Color"/><br>Mpo Score 6 Color: <input type="color" name="mpo6Color"/><br>');
-          } else {
-            document.write('Display Color: <input type="color" name="mpo0Color"/><br>');
-          }
-        </script>
-
-        Fingerprint Type:<br/>
-        <input type="radio" name="fp_radio" id="morgan" value="morgan">
-          <a target="_blank" href="https://www.rdkit.org/docs/GettingStartedInPython.html#morgan-fingerprints-circular-fingerprints">Morgan</a>
-        </input><br/>
-        <input type="radio" name="fp_radio" id="atom-pair" value="atom-pair">
-          <a target="_blank" href="https://www.rdkit.org/docs/GettingStartedInPython.html#atom-pairs-and-topological-torsions">Atom Pair</a>
-        </input><br/>
-        <input type="radio" name="fp_radio" id="fpm" value="fpm">
-          <a target="_blank" href="https://www.rdkit.org/docs/source/rdkit.Chem.Fingerprints.FingerprintMols.html#rdkit.Chem.Fingerprints.FingerprintMols.FingerprintMol">Fingerprint Mol</a>
-        </input><br>
-
-        <script>
-          function onReclusterSingletonsCheck() {
-            const checkbox = document.getElementById('reclusterCoefficientCheck');
-            console.log(checkbox)
-            if(checkbox.checked) {
-              document.getElementById('reclusterCoefficient').classList.remove('hidden') 
-              document.getElementById('reclusterCoefficient').style.visibility = "visible";
-              console.log(document.getElementById('reclusterCoefficient'));
-            } else {
-              document.getElementById('reclusterCoefficient').classList.remove('hidden') 
-              document.getElementById('reclusterCoefficient').style.visibility = "hidden";
+         <style> 
+            .gradient { 
+            font: "Roboto"; 
+            text-align: center; 
+            background: linear-gradient(to right, 
+            #0000ff, #add8e6); 
+            } 
+         </style>
+         <b>Current Color for mpo0:</b> 
+         <input class="color1" type="color" name="mpo0color" value="#0000ff" /> 
+         <br>
+         <b>Current Color for mpo6:</b> 
+         <input class="color2" type="color" name="mpo6color" value="#add8e6" /> 
+         <div class="gradient">
+            <b>Color Gradient</b>
+            <script> 
+               var color1 = document.querySelector(".color1"); 
+               var color2 = document.querySelector(".color2"); 
+               var body = document.querySelector(".gradient"); 
+               
+               // Changing color for the gradient 
+               function changeGradient() { 
+                 body.style.background = 
+                       "linear-gradient(to right, "  
+                           + color1.value + ", "  
+                           + color2.value + ")"; 
+               } 
```

Comment:
Is there any way we could make this slightly smaller and next to the color selectors

Reply:
![image](https://user-images.githubusercontent.com/17461643/92940958-c552fa80-f41d-11ea-8b6d-5dd513f95254.png)

Is this better? (Also in latest commit)

Reply:
Any chance we could cut the size of the gradient in half again? I just feel like it's super dominant lol